contributor-guides/for-individuals.md

```diff
@@ -5,58 +5,77 @@ explains: How individuals can help the Foundation for Public Code
+It contains all of its documentation, procedures and information as well as explanations on what the Foundation for Public Code does.
+We see it as a living charter.
 
 In this guide we try to set out all the ways you can contribute to the Foundation for Public Code.
 
 ## How you can help
 
-Outlined below are some ways you can help. Please note that this project is released with a [Contributor Code of Conduct](../CODE_OF_CONDUCT.md). By participating in this project you agree to abide by its terms. Please be lovely to all other community members.
+Outlined below are some ways you can help.
+Please note that this project is released with a [Contributor Code of Conduct](../CODE_OF_CONDUCT.md).
+By participating in this project you agree to abide by its terms.
+Please be lovely to all other community members! :heart: :rainbow:
 
-### Advise us and help us solve issues
+### Contribute to open issues
 
-Please look at [the list of issues marked 'help wanted'](https://github.com/publiccodenet/about/labels/help%20wanted). These are all of the issues (and proposed changes to content) where we feel we need help.
+You can find a list of open issues on [our backlog](https://github.com/publiccodenet/about/issues).
 
-If you respond to an issue you will be notified by email of any new responses. You can reply to that email to reply in the thread. If you are interested in a certain discussion but do not want to post, use the 'subscribe' button.
+Issues are best solved by collaboration, so don't worry if you don't know all the answers!
+We'd love to get your comments, perspectives and encouragements 👍.
 
-### Identify issues we need to solve
+If you respond to an issue you will be notified by email of any new responses.
+You can reply to that email to reply in the thread.
+If you are interested in a certain discussion but do not want to post, use the 'subscribe' button.
 
-If you have any suggestion about our organization, improvements to our documentation and communications or questions we invite you to [file an issue](https://github.com/publiccodenet/about/issues/new).
+### Identify new issues
 
-For instance you could give us feedback on this guide, the [README](../README.md) or any other file in this repository.
+Let us know what's important to you [by creating an issue](https://github.com/publiccodenet/about/issues/new).
+This can be something that's not clear, is missing and you'd like to see in the future, or something you found useful and could be expanded.
+This could be anything related to the Foundation for Public Code, including our organization, our documentation and/or our communications.
 
 Resolving these issues is a community effort as well, so be sure to clearly describe what you mean as others might not have the same context that you have.
 
-If your issue is a question then follow the good survey question rules:
+Here are some general tips for making a good issue:
```

Comment:
This repeats content from activities/trainings/writing-issues.md ... would it be better to link that?

(Being able to include fragments is one of the strengths of reStructuredText, if we find ourselves leaning in the direction of wanting to have content duplication, that would be a reason to look at `.rst` or other markup.)

Reply:
Good catch @ericherman! 

The intention was originally to move the bullet points into writing-issues.md. It then seemed good to leave some general tips in contributing.md, so it was reworked slightly. Reviewing this now, I feel we can cut the section from writing-issues.md. Issues are not surveys, so it's a bit out of place. The `short and clear` bullet was already touched upon in the heading above. I have gone ahead and turned the two remaining bullets into a simple sentence.